dev/com.ibm.ws.jaxrs.2.0.client_fat/bnd.bnd

```diff
@@ -39,6 +39,13 @@ src: \
 
 fat.project: true
 
+# These features get added programatically
+tested.features: \
+  jaxrsclient-2.1,\
+  jaxrs-2.1,\
```

Comment:
again, if we are repeating the tests with different JAX-RS levels, should we take the version number out of the bucket name?